client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -51,67 +53,81 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
             }}
-            name="username"
-            autoComplete="username"
-            autoFocus
-            helperText={touched.username ? errors.username : ''}
-            error={touched.username && Boolean(errors.username)}
-            value={values.username}
+            placeholder="Your email"
+            name="email"
+            autoComplete="email"
+            helperText={touched.email ? errors.email : ''}
+            error={touched.email && Boolean(errors.email)}
+            value={values.email}
             onChange={handleChange}
           />
+          <Typography className={classes.label}>Name</Typography>
           <TextField
-            id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
+            variant="outlined"
+            color="secondary"
+            id="username"
             fullWidth
-            margin="normal"
+            margin="dense"
             InputLabelProps={{
               shrink: true,
             }}
             InputProps={{
               classes: { input: classes.inputs },
             }}
-            name="email"
-            autoComplete="email"
-            helperText={touched.email ? errors.email : ''}
-            error={touched.email && Boolean(errors.email)}
-            value={values.email}
+            placeholder="Your name"
+            name="username"
+            autoComplete="username"
+            autoFocus
+            helperText={touched.username ? errors.username : ''}
+            error={touched.username && Boolean(errors.username)}
+            value={values.username}
             onChange={handleChange}
           />
+          <Typography className={classes.label}>Password</Typography>
           <TextField
+            variant="outlined"
+            color="secondary"
             id="password"
-            label={<Typography className={classes.label}>Password</Typography>}
             fullWidth
-            margin="normal"
+            margin="dense"
             InputLabelProps={{
               shrink: true,
             }}
             InputProps={{
               classes: { input: classes.inputs },
             }}
+            placeholder="Create a password"
             type="password"
             autoComplete="current-password"
             helperText={touched.password ? errors.password : ''}
             error={touched.password && Boolean(errors.password)}
             value={values.password}
             onChange={handleChange}
           />
-
           <Box textAlign="center">
-            <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
-              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Create'}
+            <Button type="submit" size="large" variant="contained" color="secondary" className={classes.submit}>
+              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'SIGN UP'}
```

Comment:
Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.

Reply:
I see. I will remember this issue.